cmd/server/main.go

```diff
@@ -207,6 +208,8 @@ func main() { //nolint: funlen
 	}
 
 	{ // dynamic uri handler: webhook uri is retrieved from request.URL
+		var validWebhook = regexp.MustCompile(`^[a-z0-9]+\.webhook\.office\.com/webhookb2/[a-z0-9\-]+@[a-z0-9\-]+/IncomingWebhook/[a-z0-9]+/[a-z0-9\-]+$`)
```

Comment:
Oh, I am sorry @kbudde I completely forgot to reply. 
I still don't know whether we should have that "dependency". How about making it optional by providing a flag `--validate-webhook-url` which is set to `false` by default?

Reply:
Ok, I think I know how to do it.
I'll add a regex for webhook validation as config flag.
Default value: .+ 
So by default it will work without config but it is possible to enforce a very strict check as well.
I'll add examples for old and new webhook format.  Ok?

Reply:
I would rather make it easy and simply provide a boolean input flag. If true, then use the strict regex, the one you implemented first. If there is a need for other regexes, we can discuss it further in another issue. Sounds good?

Reply:
I just want to keep it stupid simple.